specs/pvs/corla.pvs

```diff
@@ -636,9 +767,10 @@ BEGIN
 % Note that https://github.com/usnistgov/ElectionResultsReporting/ defines
 % it as "Undervote: Occurs when the voter does not select a candidate in
 % a 1-of-M contest or selects fewer than N candidates in an N-of-M contest."
-% @todo kiniry Should we model a blank vote separately from an
-% undervote?  Or do they simply use the term blank vote to denote an
-% undervote? @trace CDOS Audit Board Utility Mock-up 21 July 2017
+%%% \todojrk{Should we model a blank vote separately from an
+%%% undervote?  Or do they simply use the term blank vote to denote an
+%%% undervote?}
```

Comment:
As explained in a email from Dwight 2017-07-27, they use the term "blank vote" for zero votes, and "undervote" for "less than votes_allowed votes".